index.html

```diff
@@ -10,42 +10,178 @@
 </head>
 
 <body>
```

Comment:
- [ ] [OPTIONAL] In the body of your design, there is an excess space you need to remove so that your design can look more like the provided template design.

### Youru Design
![Screenshot from 2022-03-20 05-08-07](https://user-images.githubusercontent.com/58692590/159147762-d61cfcbe-a8f5-4005-8ecc-33a8a78accc9.png)